scripts/open-coredump.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+
+# Script for "one-click" opening of coredumps.
+# It extracts the build-id from the coredump, retrieves metadata for that
+# build, downloads the binary package, the source code and finally
+# launches the dbuild container, with everything ready to load the coredump.
+# The script is reentrant: running it after the prepartory steps will re-use what
+# is already donwloaded. It is *strongly* recommended to run this from an empty
+# directory, with nothing but the core-file present.
+#
+# Requirements:
+# * eu-unstrip
+# * curl
+# * jq
+# * git
+
+set -e
+
+SCYLLA_S3_RELOC_SERVER_URL="http://backtrace.scylladb.com"
```

Comment:
`SCYLLA_S3_RELOC_SERVER_URL=${SCYLLA_S3_RELOC_SERVER_URL:-http://backtrace.scylladb.com}`
would allow overriding it.

Reply:
You mean by an env variable?
That would be a good idea. I had to change this for testing (to point to local instance of s3 reloc server).

Reply:
yes